mobile/salto/src/routes/components/WadHistoryModal.svelte

```diff
@@ -0,0 +1,304 @@
+<script lang="ts">
```

Comment:
You have both 
mobile/salto/src/routes/components/WadHistoryModal.svelte
and 
mobile/salto/src/routes/components/WadHistoryPage.svelte

I believe one must be deleted